Modules/Core/Common/src/itkTBBMultiThreader.cxx

```diff
@@ -137,7 +137,6 @@ namespace
 {
 struct TBBImageRegionSplitter : public itk::ImageIORegion
 {
-  static const bool is_splittable_in_proportion = true;
```

Comment:
I think we need this, because TBB checks its existence.
https://software.intel.com/en-us/node/506143

Reply:
It is better to have TBB call our proportional split, then to emulate proportional splitting by repeated calls to our 1:1 split.

Reply:
@dzenanz I'm splitting this into two PR's. This PR is now only the single header fix.

Regarding the TBB, I don't know this behavior of TBB well enough to answer your question.
I'm added a proposal in #826, but it is not very satisfying.  Your advice would be greatly appreciated.

Reply:
I saw this comment after closing this PR. #826 also includes the header fix. Feel free to split and reopen if you wish.